projectatomic/ai-studio
# [wip] Fix pulled application status

### What does this PR do?

This PR merges the Environment manager into the Application manager. 

### Screenshot / video of UI



### What issues does this PR fix or reference?

Fixes #270


### How to test this PR?

Play with environments (remove, restart) or pods (remove) and check if the status is corrected reflected in the recipe page.
